scripts/docker/docker-compose-template.yml

```
@@ -0,0 +1,46 @@
+version: "3"
+services:
+    db:
+        image: mysql:5.7
+        volumes:
+            - db-data:/var/lib/mysql
+        environment:
+            MYSQL_ROOT_PASSWORD: change_me
+            MYSQL_DATABASE: opendoorstempsite
+            MYSQL_USER: db_user
+            MYSQL_PASSWORD: change_me
+        ports:
+            - "3307:3307"
```

If the MySQL inside is running on its default port of 3306, this needs to be 
```suggestion
            - "3307:3306"
```